main/src/ca/uwaterloo/flix/language/phase/Weeder2.scala

```diff
@@ -1190,6 +1191,12 @@ object Weeder2 {
       }
     }
 
+    private def visitLambdaPlus(tree: Tree)(implicit sctx: SharedContext): Validation[Expr, CompilationMessage] = {
+      val loc = tree.loc
+      val e = Expr.Ambiguous(Name.mkQName("Add.add", loc), loc)
```

Comment:
In the full version I refactor and introduce some map from operators to strings.